api/filter/http/accesslog.proto

```diff
@@ -0,0 +1,110 @@
+syntax = "proto3";
```

Comment:
Should this go somewhere else? I think a lot of this will get reused for TCP proxy access logging. Perhaps just in api/filter? cc @ggreenway

Reply:
Discussed with @ggreenway in Slack. I'm going to move the config up to api/filter.

Reply:
👍